tool.go

```diff
@@ -771,7 +771,7 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 		allArgs = []string{"--require", "source-map-support/register"}
 		if err := exec.Command("node", "--require", "source-map-support/register", "--eval", "").Run(); err != nil {
 			if !quiet {
-				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Use Node.js 4.x with source-map-support module for nice stack traces.")
+				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Install source-map-support module for nice stack traces. See https://github.com/gopherjs/gopherjs#installation-and-usage.")
```

Comment:
Well, after looking at it more, I thought making a small subsection could work well after all.

Take a look at https://github.com/gopherjs/gopherjs/pull/629/commits/4e30b5aa07e19a395bb551c01bc7607a8b0bffbc. How's it look?